Packetery/Checkout/Setup/InstallSchema.php

```diff
@@ -289,6 +289,15 @@ public function pricingRulesTable(SchemaSetupInterface &$setup): void
                     'after' => 'carrier_id'
                 ]
             ],
+            'max_cod' => [
+                "type" => Table::TYPE_DECIMAL,
+                'attr' => [
+                    'nullable' => true,
+                    'after' => 'address_validation',
+                    'length' => '20,4',
+                    'comment' => 'From what cart value will be COD payment methods hidden?'
```

Comment:
Je v pořádku, že k tomu nikde nevidím překlad? Jinak bych formuloval spíše "the value from which COD payment methods will be hidden" nebo možná ještě lépe "minimal value to hide COD payment methods".